### ISSUE TITLE
Debug Toolbar Panels Not working on Brave Browser

### ISSUE THREAD

**Issue by adamlh** [jazzband/django-debug-toolbar#1261]
Tried turning the Brave Browser Shield off but none of the panels are clickable and navigate to the relevent information. Any idea if theres a simple fix my end as it works perfectly in Chrome.

Uncaught TypeError: Failed to execute 'getComputedStyle' on 'Window': parameter 1 is not of type 'Element'.
    at Object.visible (VM14 toolbar.js:25)
    at HTMLAnchorElement.<anonymous> (VM14 toolbar.js:72)
    at HTMLUListElement.<anonymous> (VM14 toolbar.js:7)

toolbar.js

![image](https://user-images.githubusercontent.com/1850328/79767280-06d4f300-8321-11ea-8841-2b01f008ca12.png)

Seems like the type is not an Element and is throwing this error when using Brave but not when using chrome.

**Comment by paolodina** [comment]
I didn't try with Brave, but I have the same exact error with Chrome (80.0.3987.116) and Firefox (68.5.0esr) as well. It makes the toolbar unusable (panels doesn't open on click).

I noted that everything works OK on admin pages.

**Comment by paolodina** [comment]
BUT, I also see that running the demo project in the `example` subdirectory of this repo, everything works.

**Comment by paolodina** [comment]
@adamlh, note this screenshot. In the example project the value is just `VersionsPanel`, and further in my code (and I guess in yours too) there is no element with id `VersionsPanel focus--mouse`. So I get `current = null` and then the issue. I think there is nothing wrong with django debug toolbar, but probably we should have a look at something else imported which is the reason for the issue. It's still weird that Chrome works for you and not for me.

![Screenshot_2020-04-29_10-31-43](https://user-images.githubusercontent.com/1157401/80576037-c8b48f00-8a04-11ea-9a1c-9eb0b5c325f7.png)

**Comment by adamlh** [comment]
I'm using Chrome version 81.0.4044.129 and its working.

![image](https://user-images.githubusercontent.com/1850328/80593234-be9c8b80-8a18-11ea-9ee7-63fea4c2ea3b.png)


Found on Brave that it was an extension that I had enabled.
![image](https://user-images.githubusercontent.com/1850328/80593562-5b5f2900-8a19-11ea-914a-a86597ac42d9.png)

Disabled the extension and was not working
![image](https://user-images.githubusercontent.com/1850328/80593610-716ce980-8a19-11ea-9d86-18125f5bd85b.png)

Refreshed and it was now working fine with Brave with the Brave Shield UP
![image](https://user-images.githubusercontent.com/1850328/80593657-7c277e80-8a19-11ea-83ba-471196274315.png)

So it looks like its the STEEM Keychain extension for me that stopped it working.
